presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4553,6 +4553,46 @@ public void testRowNumberMultipleFilters()
         assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    @Test
+    public void testRowNumberInvalidFilters()
+    {
+        // Test invalid equality filter with ORDER BY. This should create a Window Node with a Filter Node on top.
+        MaterializedResult actual1 = computeActual("" +
```

Comment:
maybe you could provide something like `assertQueryReturnEmptyResult` (similar to `assertQueryFails`)

Reply:
or you can do what other tests do:
```
assertQuery(..., "SELECT 1 WHERE FALSE");
```